File: BowlPicks.Api/Models/PickModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BowlPicks.Api.Models
{
    public class PickModel
    {
        public int GameId { get; set; }
        public bool IsTeam1Selected { get; set; }
    }
}

File: BowlPicks/app/models/PickModel.js
"use strict";

if (typeof BowlPicks == "undefined") {
    window.BowlPicks = {};
}

if (typeof BowlPicks.Models == "undefined") {
    BowlPicks.Models = {};
}

if (typeof BowlPicks.Models.PickModel == "undefined") {
    BowlPicks.Models.PickModel = function () {
        this.GameId = 0;
        this.IsTeam1Selected = false;
    };
}